### DISCUSSION TITLE
Is axys renamed to axmol?

### DISCUSSION THREAD

**Discussion by cchuicchui** [axmolengine/axmol#881] [Q&A] [answered]
Is axys renamed to axmol?

**Comment by halx99** [comment, marked as answer]
Yes, axmol  be explain as X Molecule engine

**Reply by TyelorD** [reply]
I'm curious as to what prompted the changes from ADXE, to AXYS, to AXMOL.

**Reply by DelinWorks** [reply]
quite a nice name ngl! i like it

**Comment by cchuicchui** [comment]
Nice and thanks

**Comment by aismann** [comment]
some splashscreen stuff: (only examples, maybe copyright on it)
https://www.bing.com/images/search?view=detailV2&ccid=qPrANe0b&id=3B74FD3B411695979305439D61FEFA4B6C49E07A&thid=OIP.qPrANe0bP3NNfavTyOCnSAHaHJ&mediaurl=https%3A%2F%2Ffilkaline.com%2Fwp-content%2Fuploads%2F2019%2F01%2Fconnected-molecules.jpg&exph=680&expw=704&q=x+molecule+engine&form=IRPRST&ck=71D70431A39F36B823C21F75E2F9D861&selectedindex=6&ajaxhist=0&ajaxserp=0&pivotparams=insightsToken%3Dccid_tWwBySQL*cp_A3D3C06BBD4A1DD17CFDCC8B5DD01A38*mid_C0A4F3D2C32BC7FE1EFF0AB0C7EDF4E79269D587*simid_608000492943968816*thid_OIP.tWwBySQLvMa7rFuysnmk-wHaHo&vt=0&sim=11&iss=VSI&cdnurl=https%3A%2F%2Fth.bing.com%2Fth%2Fid%2FR.a8fac035ed1b3f734d7dabd3c8e0a748%3Frik%3DeuBJbEv6%252fmGdQw%26pid%3DImgRaw%26r%3D0&simid=608000754927018654&ajaxhist=0&ajaxserp=0
![image](https://user-images.githubusercontent.com/8652787/193649490-7e237de5-60c7-41a7-85cd-2ea2df686819.png)
![image](https://user-images.githubusercontent.com/8652787/193649561-8cf3c372-f408-454f-9402-df8bd5f0d0eb.png)
![image](https://user-images.githubusercontent.com/8652787/193649681-6e65dd67-3f2b-4e3b-a576-5cebe5d498b4.png)
![image](https://user-images.githubusercontent.com/8652787/193649718-41ca64c2-c5a5-43a1-a7fc-9d2e6dff4096.png)

**Reply by paulocoutinhox** [reply]
Nice. Without background is better.

**Reply by aismann** [reply]
And animated with axmol own sources.
Position, color and size morphing

**Reply by halx99** [reply]
nice

**Reply by aismann** [reply]
Its time to change icon and splashscreen.
Maybe animated molecules?
Better a First draft as the wrong cocos2dx..

Maybe morphing from a 2D molecule to a 3D one and Back...endless

**Reply by halx99** [reply]
> Its time to change icon and splashscreen. Maybe animated molecules? Better a First draft as the wrong cocos2dx..
> 
> Maybe morphing from a 2D molecule to a 3D one and Back...endless

Yes, but needs somebody design MIT License logo for axmol

**Comment by aismann** [comment]
![IMG_20221101_111054](https://user-images.githubusercontent.com/8652787/199210664-6a9e905a-67cc-4bb4-942d-4059c1c2e6a7.jpg)
A logo is very important to place it on some Extensions Tools pages.
Here on Spine web page (Cocos2d-x)
Thats also one way to make axmol more public.

**Reply by halx99** [reply]
Yes, but we needs a logo designer

**Reply by rh101** [reply]
@halx99 A very talented graphic designer has agreed to help with creating a logo for this game engine, one which will be put under the MIT license.  Please provide a description and any samples of what you have in mind for the look of the logo, and I'll pass it all on to the graphic designer.

**Reply by halx99** [reply]
Looks like follow draft graphic, maybe the logo chould have perspective elements:


![image](https://user-images.githubusercontent.com/6977319/199897939-2428fb8e-5de8-4c2b-995a-ffe9394bfbb2.png)

**Reply by halx99** [reply]
A X Molecular Engine.

**Reply by rh101** [reply]
No worries, I'll pass this on.  If you can provide any more details, or even a perspective sketch, then that would be great. What do the different elements in there represent (the triangle, the pentagon shape and the rays coming out)?

**Reply by halx99** [reply]
The elements could contains cone and rays

**Reply by halx99** [reply]
The cone maybe construct with 3~4 parts(like molecular is constructed with atoms).

**Reply by rh101** [reply]
I understand.  All this info should help. Thanks :)

**Reply by DelinWorks** [reply]
I'm very excited on how well it will look :D good luck! and the name is very cool!

**Reply by aismann** [reply]
Any news? 
We are very excited and hoping this will be the start of 'axmol' 1.0 release;

**Comment by rh101** [comment]
@halx99 Initial artwork concepts:

Concept 1:
![image](https://user-images.githubusercontent.com/8603230/202937603-4bb9ccf4-fa44-4530-9e43-0906ba981022.png)
Concept 2:
![image](https://user-images.githubusercontent.com/8603230/202937611-486db92f-b126-4db2-9fa0-990bade02e59.png)
Concept 3:
![image](https://user-images.githubusercontent.com/8603230/202937616-13a44357-e4a7-4dbd-b714-a7df8df10236.png)
Concept 4:
![image](https://user-images.githubusercontent.com/8603230/202937630-7adece70-2bc3-4fcc-84bd-d0fe723cbed7.png)
Concept 5:
![image](https://user-images.githubusercontent.com/8603230/202937634-ea3d5f6b-216c-41d0-9bd8-0d6781a6471c.png)


See if any fit what you have in mind. If any do, and you want changes, that's fine, and if none are suitable, then that's also OK.  The designer is willing to help with getting the concept as close as possible to what you want (time permitting).

**Reply by halx99** [reply]
Nice work, all of them looks good, and I preferred the concept 5. A lilttle problem is the perspective looks incorrect, the larger(near) circle should override the smaller(far), I'm not sure does it been designed reasonable.

**Reply by rh101** [reply]
I can actually see what you mean, kind of like an optical illusion.  I'll have a chat with the designer about this, and hopefully get a new version of it soon.  So you want the smaller pentagon to be further away, correct?

**Reply by halx99** [reply]
> I can actually see what you mean, kind of like an optical illusion. I'll have a chat with the designer about this, and hopefully get a new version of it soon. So you want the smaller pentagon to be further away, correct?

Correct.

**Comment by rh101** [comment]
@halx99 Is this better (with different versions for reference):

![image](https://user-images.githubusercontent.com/8603230/202951132-f2e762b6-b58f-4e0b-a7f5-55046c22cb9b.png)
![image](https://user-images.githubusercontent.com/8603230/202951208-767fcb24-7eed-4bf1-80a7-616deea209be.png)
![image](https://user-images.githubusercontent.com/8603230/202951220-cd0317ee-21dc-4cd4-953f-d17972a94f8d.png)
![image](https://user-images.githubusercontent.com/8603230/202951232-7268f98e-4a10-494c-9ebc-39cd9eb24dc1.png)
![image](https://user-images.githubusercontent.com/8603230/202951252-7d8b33c4-a87d-49a4-8eec-5a49427240bb.png)

**Reply by halx99** [reply]
> @halx99 Is this better (with different versions for reference):
> 
> ![image](https://user-images.githubusercontent.com/8603230/202951132-f2e762b6-b58f-4e0b-a7f5-55046c22cb9b.png) ![image](https://user-images.githubusercontent.com/8603230/202951208-767fcb24-7eed-4bf1-80a7-616deea209be.png) ![image](https://user-images.githubusercontent.com/8603230/202951220-cd0317ee-21dc-4cd4-953f-d17972a94f8d.png) ![image](https://user-images.githubusercontent.com/8603230/202951232-7268f98e-4a10-494c-9ebc-39cd9eb24dc1.png) ![image](https://user-images.githubusercontent.com/8603230/202951252-7d8b33c4-a87d-49a4-8eec-5a49427240bb.png)

Yes, it's better.

**Reply by rh101** [reply]
If you're happy with the logo, then the only thing left to pass on to the designer is which format you want the files to be saved as.

**Reply by halx99** [reply]
I think .png, .ico required.

**Reply by halx99** [reply]
And, maybe the design source file such as .psd also should been provide since the logo is MIT licensed.

**Reply by DelinWorks** [reply]
yeah, it would be nice to have a .psd file because then you can put any solid color background without losing the original design.

**Reply by halx99** [reply]
What about make the word gradient? just a idea, not force required.

**Reply by rh101** [reply]
> What about make the word gradient? just a idea, not force required.

Sure, I'll pass that suggestion on, along with the info regarding the design source file and output formats.

**Reply by aismann** [reply]
@halx99 
It looks we have an AXMOL 1.0 release in some days?

**Reply by DelinWorks** [reply]
the .ico can be the design only without the letters it would look 👌

**Comment by rh101** [comment]
This is the file in SVG format, so you load it up in any application (such as Adobe Illustrator, Inkscape etc) and edit or export the logo:

![AmolLogo](https://user-images.githubusercontent.com/8603230/203264608-16a1d986-55ad-452d-b4f4-2ac74e459dfd.svg)

@halx99 Shading can be added to the text easily in any of the vector editors (Inkscape works great). Just select the text, and then select the gradient tool and go from there.

The original vector graphic file was is in PDF format (PDF is just a wrapper around the vector data), but github wouldn't allow me to attach it to a post on here, so I loaded it up in Inkscape, and re-saved it as SVG format.

**Comment by aismann** [comment]
Any news there?

**Reply by rh101** [reply]
The logo is being used on the [front page](https://github.com/axmolengine)
